test/config-luatest/console_test.lua

```diff
@@ -58,11 +58,8 @@ local function assert_console_works(cluster, config)
     t.assert_equals(eval(s, 'return 123'), 123)
 end
 
-local function assert_console_closed(_replicaset, _config)
-    -- TODO (gh-9535): Enable it after a fix in the console
-    -- applier. Now it doesn't stop old console sockets.
-    --
-    -- t.assert_error(connect, cluster, 'instance-001', config, 0.1)
+local function assert_console_closed(_replicaset, config)
+    t.assert_error(connect, cluster, 'instance-001', config, 0.1)
```

Comment:
It always passes, because the instance name is `instance-001` instead of `i-001`.

Reply:
Please, leave comments open: usually a person, who opens the discussion, resolves it.

Reply:
Done